cypress/integration/buy-shirt.spec.ts

```diff
@@ -1,25 +1,70 @@
+import { MenuContentPage } from "../page/index"
+import { ProductListPage } from "../page/index"
+import { ShoppingCartPage } from "../page/index"
+import { LoginPage } from "../page/index"
+import { ShippingStepPage } from "../page/index"
+import { AddressStepPage } from "../page/index"
+import { PaymentStepPage } from "../page/index"
+
+
+const menuContentPage = new MenuContentPage();
+const productListPage = new ProductListPage();
+const shoppingCartPage = new ShoppingCartPage();
+const loginPage = new LoginPage();
+const shippingStepPage = new ShippingStepPage();
+const addressStepPage = new AddressStepPage();
+const paymentStepPage = new PaymentStepPage();
+
+
 describe("Buy a t-shirt", () => {
 
     it("then the t-shirt should be bought", () => {
-        cy.visit("http://automationpractice.com/");
-        cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
-        cy.get("#center_column a.button.ajax_add_to_cart_button.btn.btn-default").click();
-        cy.get("[style*=\"display: block;\"] .button-container > a").click();
-        cy.get(".cart_navigation span").click();
 
-        cy.get("#email").type("aperdomobo@gmail.com");
-        cy.get("#passwd").type("WorkshopProtractor");
+        menuContentPage.visitMenuContentPage()
+        //cy.visit("http://automationpractice.com/");
+        menuContentPage.goToTShirtMenu()
+        //cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
+
+        //product
+        productListPage.goAddToCart()
+        //cy.get("#center_column a.button.ajax_add_to_cart_button.btn.btn-default").click();
+        productListPage.goThisButton()
+        //cy.get("[style*=\"display: block;\"] .button-container > a").click();
 
-        cy.get("#SubmitLogin").click();
-        cy.get(".cart_navigation span").click();
+        //shopping
+        shoppingCartPage.Cart()
+        //cy.get(".cart_navigation span").click();
 
-        cy.get(".checker").click();
-        cy.get(".cart_navigation span").click();
+        //login
+        loginPage.insertMail()
+        //cy.get("#email").type("aperdomobo@gmail.com");
+        loginPage.insertPassword()
```

Comment:
receive the password as param of the insertPassword function

Reply:
Done